app/components/meeting/edit-meeting-modal.js

```diff
@@ -0,0 +1,99 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { task } from 'ember-concurrency';
+import EmberObject from '@ember/object';
+import { A } from '@ember/array';
+import CONFIG from 'frontend-kaleidos/utils/config';
+
+/**
+ * @argument {meeting}
+ * @argument {didSave}
+ * @argument {onCancel}
+ */
+export default class MeetingEditMeetingComponent extends Component {
+  @service toaster;
+
+  @tracked kind;
+  @tracked selectedKindUri;
+  @tracked startDate;
+  @tracked extraInfo;
+  @tracked _meetingNumber;
+  @tracked _numberRepresentation;
+
+  meetingYear = this.meeting.plannedStart.getFullYear();
+
+  constructor() {
+    super(...arguments);
+
+    this.selectedKindUri = this.meeting.kind;
+    this.kind = EmberObject.create(
+      CONFIG.MINISTERRAAD_TYPES.TYPES.find(
+        (minsterraad) => minsterraad.uri === this.selectedKindUri
+      )
+    );
+    this.startDate = this.meeting.plannedStart;
+    this.extraInfo = this.meeting.extraInfo;
+    this.meetingNumber = this.meeting.number;
+    this.numberRepresentation = this.meeting.numberRepresentation;
+  }
+
+  get date() {
+    return A([this.startDate]);
+  }
+
+  get meeting() {
+    return this.args.meeting;
+  }
```

Comment:
Hier ben ik eerlijk gezegd niet zo'n fan van, omdat het verbergt dat `meeting` een argument is van de component.